ui/app/components/app/connected-accounts-list/connected-accounts-list-options-item/connected-accounts-list-options-item.component.js

```diff
@@ -0,0 +1,27 @@
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+
+export default class ConnectedAccountsListOptionsItem extends PureComponent {
+  static propTypes = {
+    iconClassNames: PropTypes.string,
+    children: PropTypes.node.isRequired,
+    onClick: PropTypes.func,
+  }
+
+  static defaultProps = {
+    iconClassNames: undefined,
+    onClick: undefined,
+  }
+
+  render () {
+    const { children, iconClassNames, onClick } = this.props
+
+    return (
+      <div className="connected-accounts-options__row" onClick={onClick}>
+        <i className={classnames('connected-accounts-options__row-icon', iconClassNames)} />
+        <span>{children}</span>
```

Comment:
Nit: Does this make sense as a `node`? e.g. could this be passed in as a string, instead of an arbitrary node via `children`?

Where it is being put in a `<span>`.... not that you can't put non-text in a span, but it does seem to be meant for text.

Reply:
This seems fine to me, should we want to format the text

Reply:
Sure, fair enough.